client/src/components/Results.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import queryString from 'query-string';
+import { Link } from 'react-router-dom';
+import { Button } from 'react-bootstrap';
+
+import Spinner from './Spinner';
+import MapElement from './MapElement';
+import './Results.scss';
+import Listings from './Listings';
+
+
+class Results extends React.Component {
+  // This contains the definitions of the types for the props
+  // this component needs
+  static propTypes = {
+    location: PropTypes.shape({
+      search: PropTypes.string,
+    }),
+  };
+
+  static ascendingSortHelper(a, b) {
```

Comment:
Since you've refactored all of these functions into the Listings component, these functions are not needed here anymore (ascendingSortHelper, descendingSortHelper, sortAscending, sortDescending)